frontend/locales/en/strings.json

```diff
@@ -803,6 +803,14 @@
       "title": "Official images",
       "description": "The official list of tested Amazon Machine Images (AMIs) that are automatically used to create AWS ParallelClusters."
     },
+    "helpPanel": {
+      "title": "Official images",
+      "description": "<p>Refer to this list of AMI images when configuring your cluster.</p><p>This version-coupled instance of the $t(global.projectName) automatically uses these tested AMIs to create AWS ParallelClusters based on the operating system and architecture that you choose.</p>",
```

Comment:
typo: ParallelCluster**s**?

Reply:
Checking

Reply:
I checked with copywriters. We're changing it to just "clusters"